test/node.js

```diff
@@ -179,11 +185,15 @@ describe('filter addrs', () => {
     const mh2 = multiaddr('/ip4/127.0.0.1/udp/9090')
     const mh3 = multiaddr('/ip4/127.0.0.1/tcp/9090/ws')
     const mh4 = multiaddr('/ip4/127.0.0.1/tcp/9090/ws/ipfs/Qmb6owHp6eaWArVbcJJbQSyifyJBttMMjYV76N2hMbf5Vw')
+    const mh5 = multiaddr('/dns/ipfs.io/ws')
+    const mh6 = multiaddr('/dns/ipfs.io/ws/ipfs/Qmb6owHp6eaWArVbcJJbQSyifyJBttMMjYV76N2hMbf5Vw')
```

Comment:
Well, actually, default for just `/ws` is port 80 and `/wss` is port 443, that's it.

@dryajov still up to finish this? It would be sweet for the 0.23 release! :)